content/raw/chat/group_chat.md

```diff
@@ -0,0 +1,50 @@
+# Group chat
+Group chats are meant to be private ad-hoc groups.
+
+# Rules
+
+### Glossary
+Group creator - one who creates new group  
+Member - anyone in the group  
+Admin - group administrator  
+User - Member who is not an Admin  
+
+## Group creation
+Group creator is only allowed to add mutual contacts. This is important, both for privacy and as a spam prevention measure.
+- Group chat MUST contain at least one Member
```

Comment:
What happens if the last user of the group is removed? Should it be allowed?

Reply:
This applies only to `Group creation` stage, the group can be left by everyone and become a dead "Zombie" group.